src/utils/apiRequests/api.ts

```diff
@@ -37,11 +37,25 @@ export async function getRequest(url: any): Promise<any> {
       result = res.status === 200 ? await res.json() : null;
       if (res.status === 404) {
         const errorMessage = 'Not Found';
```

Comment:
Let's use a translation resource for `Not Found`.

Reply:
Done